app/assets/stylesheets/vendor/base/extends/_button.scss

```diff
@@ -4,15 +4,21 @@
   border-radius: $base-border-radius;
   color: white;
   display: inline-block;
-  font-size: $base-font-size;
+  font-size: $button-font-size;
   font-weight: bold;
-  line-height: 1;
-  padding: 0.75em 1em;
+  line-height: 1.5;
+  margin: auto;
+  padding: 0.9em 1.3em;
   text-decoration: none;
 
   &:hover {
     background-color: $hover-button-color;
     color: white;
+    cursor: pointer;
+  }
+
+  &:focus {
+    color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.